files/bootstrap.sh

```diff
@@ -152,6 +152,14 @@ DEFAULT_CONTAINER_RUNTIME=dockerd
 if is_greater_than_or_equal_to_version $KUBELET_VERSION "1.24.0"; then
   IS_124_OR_GREATER=true
   DEFAULT_CONTAINER_RUNTIME=containerd
+else
+  # Ensure that these exist for testing purposes and 1.21 and below
```

Comment:
I'm not following, seems like this downgrade of the API versions should happen for 1.22 and 1.23 only; can we just add an `elif` for that?

Reply:
Ya, I can do that.